conformance/image.go

```diff
@@ -35,6 +35,9 @@ type Descriptor struct {
 	// Size specifies the size in bytes of the blob.
 	Size int64 `json:"size"`
 
+	// Data specifies the data of the object described by the descriptor.
+	Data []byte `json:"data"`
```

Comment:
from the referenced commit 
```
	// Data is an embedding of the targeted content. This is encoded as a base64
 	// string when marshalled to JSON (automatically, by encoding/json). If
 	// present, Data can be used directly to avoid fetching the targeted content.
 	Data []byte `json:"data,omitempty"`
```

don't see how Data could be empty from this test bucket..  just a nit...

Reply:
It doesn't even seem to matter either way. An empty `[]byte` with and without the struct tag results in `{}` in my tests in the Go Playground.